### DISCUSSION TITLE
"Error: unknown shorthand flag: 'p' in -p" when try to run `postal initialize` command

### DISCUSSION THREAD

**Discussion by yuri-ccp** [postalserver/postal#2860] [Help with using Postal] [answered]
The title is self explanatory. I getting this error when I try to run the `postal initialize` command during the installation process:
```shell
[root@lc1 ~]# postal initialize
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Error: unknown shorthand flag: 'p' in -p
See 'podman --help'
```

Someone know something about this and can give me a light of how to solve this?

**Comment by adamcooke** [comment, marked as answer]
The error is from `podman`. We don't support this with the `postal` command.

**Comment by kdekooter** [comment]
Upgrade to the lasest docker version fixed things for me.
